### DISCUSSION TITLE
Get timestamp sequentially

### DISCUSSION THREAD

**Discussion by pless84** [faker-js/faker#2633] [Q&A] [answered]
Is it possible to get timestamps in sequence?

**Comment by ST-DDT** [comment]
Could you elaborate your request? I'm not sure what you are asking.
Maybe faker.date.betweens can help you?

**Reply by pless84** [reply]
faker.date.betweens doesn't give dates in sequence, we need it for Mockoon to give a sequence of random data in order of dates that are also generated

**Reply by matthewmayer** [reply, marked as answer]
`faker.date.betweens` then sort the results?

Or `faker.date.soon` and each time pass the previously generated date as refDate?

**Reply by ST-DDT** [reply]
Isnt faker.date.betweens sorted?

**Reply by matthewmayer** [reply]
You are right. It is sorted.
